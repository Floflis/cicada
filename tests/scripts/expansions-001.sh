echo {3..5}
# overflow, prints its original.
echo {99999999999999999999999..999999999999999999999999}
