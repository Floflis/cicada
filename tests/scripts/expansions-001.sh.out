3 4 5
{99999999999999999999999..999999999999999999999999}
